### ISSUE TITLE
Cant update Supervisor to 2023.11.1

### ISSUE THREAD

**Issue by jensjakobandersen** [home-assistant/supervisor#4703]
### Describe the issue you are experiencing

When I try to update to Supervisor 23.11.0 I get an error.

Error: Update of Supervisor failed: Can't install ghcr.io/jhome-assistant/aarch64-hassio-supervisor:2023.11.0: 404 Client Error for http+docker://localhost/v1.43/image/ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0/json: Not found ("No such image: ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0")

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Steps to reproduce the issue

Either run the update from the webpage - or open terminal and run ha supervisor update

### Anything in the Supervisor logs that might be useful for us?

```txt
23-11-12 13:35:53 INFO (MainThread) [__main__] Running Supervisor
23-11-12 13:35:53 INFO (MainThread) [supervisor.os.manager] Rauc: B - marked slot kernel.1 as good
23-11-12 13:35:53 INFO (MainThread) [supervisor.supervisor] Fetching AppArmor profile https://version.home-assistant.io/apparmor_stable.txt
23-11-12 13:35:53 INFO (MainThread) [supervisor.host.apparmor] Adding/updating AppArmor profile: hassio-supervisor
23-11-12 13:35:54 INFO (MainThread) [supervisor.supervisor] Update Supervisor to version 2023.11.0
23-11-12 13:35:54 INFO (MainThread) [supervisor.docker.interface] Downloading docker image ghcr.io/home-assistant/aarch64-hassio-supervisor with tag 2023.11.0.
23-11-12 13:35:58 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state startup
23-11-12 13:35:58 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-11-12 13:35:58 WARNING (MainThread) [supervisor.jobs] 'StoreManager.reload' blocked from execution, supervisor needs to be updated first
23-11-12 13:36:03 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-11-12 13:36:13 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
23-11-12 13:36:16 ERROR (MainThread) [supervisor.docker.interface] Can't install ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0: 404 Client Error for http+docker://localhost/v1.43/images/ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0/json: Not Found ("No such image: ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0")
23-11-12 13:36:16 INFO (MainThread) [supervisor.resolution.module] Create new issue update_failed - supervisor / None
23-11-12 13:36:16 ERROR (MainThread) [supervisor.supervisor] Update of Supervisor failed: Can't install ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0: 404 Client Error for http+docker://localhost/v1.43/images/ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0/json: Not Found ("No such image: ghcr.io/home-assistant/aarch64-hassio-supervisor:2023.11.0")
23-11-12 13:36:16 CRITICAL (MainThread) [supervisor.core] Can't update Supervisor! This will break some Add-ons or affect future versions of Home Assistant!
23-11-12 13:36:16 INFO (MainThread) [supervisor.addons] Phase 'initialize' starting 0 add-ons
23-11-12 13:36:16 INFO (MainThread) [supervisor.addons] Phase 'system' starting 0 add-ons
23-11-12 13:36:16 INFO (MainThread) [supervisor.addons] Phase 'services' starting 1 add-ons
23-11-12 13:36:16 INFO (SyncWorker_3) [supervisor.docker.manager] Cleaning addon_core_deconz application
23-11-12 13:36:18 INFO (MainThread) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-deconz with version 6.20.0
```

## System Information

version | core-2023.10.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.5
os_name | Linux
os_version | 6.1.56
arch | aarch64
timezone | Europe/Copenhagen
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.33.0
Stage | running
Available Repositories | 1400
Downloaded Repositories | 6

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | December 12, 2023 at 01:00
relayer_connected | true
relayer_region | eu-central-1
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | false
remote_server | eu-central-1-6.ui.nabu.casa
certificate_status | ready
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.0
-- | --
update_channel | stable
supervisor_version | supervisor-2023.10.1
agent_version | 1.6.0
docker_version | 24.0.6
disk_total | 28.0 GB
disk_used | 6.1 GB
healthy | failed to load: Unhealthy
supported | true
board | odroid-n2
supervisor_api | ok
version_api | ok
installed_addons | deCONZ (6.20.0), Terminal & SSH (9.7.1)

</details>

<details><summary>Dashboards</summary>

dashboards | 16
-- | --
resources | 2
views | 16
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | November 3, 2023 at 06:46
-- | --
current_recorder_run | November 12, 2023 at 19:06
estimated_db_size | 155.90 MiB
database_engine | sqlite
database_version | 3.41.2

</details>

<details><summary>Sonoff</summary>

version | 3.5.3 (a8c6d45)
-- | --
cloud_online | 7 / 9
local_online | 0 / 0
debug | failed to load: 

</details>

### Supervisor diagnostics

[config_entry-hassio-6854af4adda5d85d74a459955abf37b0.json.txt](https://github.com/home-assistant/supervisor/files/13336296/config_entry-hassio-6854af4adda5d85d74a459955abf37b0.json.txt)

### Additional information

I have also reported it here:
https://github.com/home-assistant/operating-system/issues/2920

**Comment by GJSchroeder** [comment]
I have the same issue as well. Running on Home Assistant Yellow. It happened right after I migrated from running 2023.10.5 on a Pi to the Yellow. Not sure whether that is related or just a coincidence.

**Comment by agners** [comment]
This reports the same problem as #4709. Since there is more information about this issue in #4709, I closes this one in favor of #4709.
